lib/global-styles.php

```diff
@@ -259,58 +259,68 @@ function gutenberg_experimental_global_styles_get_theme() {
  * @return array
  */
 function gutenberg_experimental_global_styles_get_block_data() {
-	// TODO: this data should be taken from the block registry.
-	//
-	// At the moment this array replicates the current capabilities
-	// declared by blocks via __experimentalLineHeight,
-	// __experimentalColor, and __experimentalFontSize.
 	$block_data = array(
-		'global'          => array(
+		'global' => array(
 			'selector' => ':root',
-			'supports' => array(), // By being blank, the 'global' section won't output any style yet.
-		),
-		'core/paragraph'  => array(
```

Comment:
Don't all these supports need to be added to the json files?

Reply:
This PR proposes to take them from the block.json, where they're already declared. However, this wasn't possible until https://github.com/WordPress/gutenberg/pull/22491 landed. So I guess the answer is: yes!